src/rpc/methods/eth/filter/mod.rs

```diff
@@ -102,6 +107,27 @@ impl EthEventHandler {
             Err(Error::msg("NotSupported"))
         }
     }
+
+    pub fn eth_new_pending_transaction_filter(&self) -> Result<FilterID, Error> {
```

Comment:
It seems similar to some previous code introduced. Can we make it more generic?

Reply:
yes updated